ljconf.go

```diff
@@ -162,6 +166,31 @@ func Load(fn string) (conf *Conf, err error) {
 	return conf, nil
 }
 
+func (c *Conf) Watch(interval time.Duration, ch chan *Conf) error {
```

Comment:
From the use case you provided. I suggest we make `Watch` a single function instead of a method of `*Conf`. The signature looks like this (comments included):

```go
// Watch periodically check the configure file in curConf with the specified interval. If the configuration file
// chnages, it's loaded and send to the specified channel as a *Conf.
func Watch(curConf *Conf, interval time.Duration, ch chan *Conf) error {
```